File: Algorithms/Sorts/JavaScript/InsertionSort.js
// Insertion Sort in JavaScript
function insertionSort(arr) {

    for (let i = 1; i < arr.length; i++) 
    {

      let curr= arr[i]
      let j
      for (j = i - 1; j >= 0 && arr[j] > curr; j--) 
      {
        arr[j + 1] = arr[j]
      }
      arr[j + 1] = curr
    }

    return arr
  }
  console.log(insertionSort([add_any_data]))

File: Algorithms/Sorts/Python/InsertionSort.py

def insertionSort(arr): 
      
    if len(arr) <= 1:
        return 
 
    for i in range(1, len(arr)):  
        key = arr[i]  
        j = i-1
        while j >= 0 and key < arr[j]:  
            arr[j+1] = arr[j]  
            j -= 1
        arr[j+1] = key  

    return arr


print(insertionSort([any_data_here]))
